var/spack/repos/builtin/packages/py-python-multipart/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPythonMultipart(PythonPackage):
+    """A streaming multipart parser for Python"""
+
+    homepage = "https://github.com/andrew-d/python-multipart"
+    pypi = "python-multipart/python-multipart-0.0.5.tar.gz"
+
+    version("0.0.5", sha256="f7bb5f611fc600d15fa47b3974c8aa16e93724513b49b5f95c81e6624c83fa43")
+
+    depends_on("python@:3.6", when="@:0.0.5", type=("build", "run"))
```

Comment:
This is causing package audits to fail on develop because we have no Python 3.6 anymore. I'm reverting the PR for now #34960, feel free to give it another try...

Reply:
Thanks, I don't know how I missed this. I don't see anything in the package that says that it requires Python 3.6 and older, so we can probably just remove this line and re-merge.

Reply:
thanks, doing that now, will reopen another PR soon.